quickjs.h

```diff
@@ -1043,6 +1043,9 @@ JS_EXTERN int JS_SetModuleExport(JSContext *ctx, JSModuleDef *m, const char *exp
 JS_EXTERN int JS_SetModuleExportList(JSContext *ctx, JSModuleDef *m,
                                      const JSCFunctionListEntry *tab, int len);
 
+JS_EXTERN void js_build_backtrace(JSContext *ctx, JSValue error_obj, JSValue filter_func,
```

Comment:
Public JS APIS use JS_CamelCase

Reply:
Not all of them: many internal APIs that have been published for special cases use lowercase with underscores.  We might want to either
- use a systematic case for all public APIs
- use a different case for lower level APIs that are not guaranteed to be stable

Reply:
The public APIs do follow the pattern. Everything we've added to NG follows it too.